src/api/dataStoreApi/helpers/desiredSortToQueryParams.ts

```diff
@@ -0,0 +1,54 @@
+import { DomainSortOptions } from "../types";
+  
+  export const domainSortingOptionsReflection = {
+    domainId: "",
+    name: undefined,
+    parent: undefined,
+    labelHash: undefined,
+    minter: undefined,
+    owner: undefined,
+    metadataUri: undefined,
+    royaltyAmount: undefined,
+    created: undefined,
+    registrar: undefined,
+    isValid: false,
+    buyNow: undefined,
+  };
+  
+  export function desiredSortToQueryParams(sort: DomainSortOptions) {
+    const possibleSortParams = Object.keys(domainSortingOptionsReflection);
+    const sortKeyValues = Object.entries(sort);
+  
+    let sortQueryString = "";
+    let sortOrderQueryString = "";
+  
+    sortKeyValues.forEach((entry) => {
+      const key = entry[0];
+      const value = entry[1];
+      if (!possibleSortParams.find((validParam) => validParam === key)) {
+        throw new Error(`Error: ${key} is not a valid sorting parameter.`);
+      }
+      if (value < 0 || value > 1) {
```

Comment:
`data-store-api` requires to pass `sortDirection` as `asc` or `desc`, not a number (0 | 1)
I think we need to define the string contants of QuerySortDirection, also make unique [direction number](https://github.com/zer0-os/data-store-api/blob/master/src/types.ts#L32) to [Sort](https://github.com/zer0-os/data-store-api/blob/master/src/types.ts#L16)